src/drivers/uavcan/CMakeLists.txt

```diff
@@ -40,7 +40,10 @@ set(UAVCAN_USE_CPP03 ON CACHE BOOL "uavcan cpp03")
 set(UAVCAN_PLATFORM "generic")
 
 if(CONFIG_ARCH_CHIP)
-	if(${CONFIG_ARCH_CHIP} MATCHES "kinetis")
```

Comment:
Forklifting (via copy&pasting code) stuff from upstream instead of taking the full commit is not a good idea as it will cause a conflict when trying to push our changes back into upstream. 

If we need this code, simply cherry-pick the commit and make a pull request to merge (rebase) it into our fork